.github/workflows/dax.yml

```diff
@@ -1,7 +1,17 @@
-# This workflow builds and tests the devdax memory provider.
-# It requires a DAX device (e.g. /dev/dax0.0) configured in the OS.
-# This DAX device should be specified using the
+#
+# This workflow builds and tests the devdax memory provider
+# and the file memory provider with FSDAX.
+# It requires:
+# - a DAX device (e.g. /dev/dax0.0) and
+# - a FSDAX device (e.g. /dev/pmem1)
+# configured and mounted in the OS.
+#
+# The DAX device should be specified using the
 # UMF_TESTS_DEVDAX_PATH and UMF_TESTS_DEVDAX_SIZE environment variables.
+#
+# The FSDAX device should be mounted (e.g. /mnt/pmem1) in the OS
+# and specified using the UMF_TESTS_FSDAX_PATH environment variable.
```

Comment:
to be precise, this PATH should point to a file on FSDAX

Reply:
Done